scripts/sommelier

```diff
@@ -217,6 +217,18 @@ start_app() {
     fi
     "${WINELOADER}" explorer /desktop="virtual","${native_res}" "${app}" "${ACTION_ARGS[@]}"
     local exit_code="$?"
+  elif [[ -f "${ACTION_ARGS}" ]]; then
+    # Passing args to wine apps
+    for i in "${ACTION_ARGS[@]}"; do
+        # Add path in Wine form (e.g. "z:/home/user/Desktop/lol.xyz")
+        if [[ -f /${i#?:} ]]; then 
```

Comment:
I'm assuming the initial `/` is to ensure the path is an absolute path? Do relative paths work without the `Z:` workaround? If not, can we turn all relative paths into absolute ones and then append `Z:` to all paths?

Reply:
Also, I think `-e` would be better in this case, so that it also works if the user is opening a directory or a socket instead of a regular file.